### PR TITLE
style(frontend): new icons for receive and convert actions in Hero [GIX-3047]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We update the Hero actions of Receive and Convert with the new set of icons provided/done by the Design team.

Screenshot 2024-10-11 at 10 34 38

### CODE REVIEW

File: src/frontend/src/eth/components/send/ConvertToCkERC20.svelte [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	import ConvertETH from '$icp-eth/components/send/ConvertETH.svelte';
 	import { ckErc20HelperContractAddress } from '$icp-eth/derived/cketh.derived';
 	import { initSendContext, SEND_CONTEXT_KEY, type SendContext } from '$icp-eth/stores/send.store';
-	import IconConvert from '$lib/components/icons/IconConvert.svelte';
+	import IconSwap from '$lib/components/icons/IconSwap.svelte';
```
Comment (peterpeterparker) [inline src/frontend/src/eth/components/send/ConvertToCkERC20.svelte:11]: Can we use the term `Swap` in the context of ck token?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/eth/components/send/ConvertToCkERC20.svelte:11]: Right! Not yet I imagine... changed to `IconCkConvert` ? WDYT?

Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/eth/components/send/ConvertToCkERC20.svelte]: done

**Review by peterpeterparker — APPROVED** [review]
LGTm, thx

**Pull request merged**